src/iter.rs

```diff
@@ -1,4 +1,6 @@
 use crate::progress_bar::ProgressBar;
+#[cfg(feature = "tokio")]
```

Comment:
If you accept the other change, this won't be necessary anymore.

Reply:
Removed